docs/requirements.txt

```diff
@@ -0,0 +1,12 @@
+recommonmark==0.6.0
+Sphinx==3.2.1
+sphinx-markdown-tables==0.0.15
+PyYAML==5.3.1
```

Comment:
I don't approve those exact version requirements. Can we use `>=` instead?

Reply:
done ;)